include/pdal/util/Bounds.hpp

```diff
@@ -189,6 +202,10 @@ class PDAL_DLL BOX3D : private BOX2D
     BOX3D()
        { clear(); }
 
+    BOX3D(const BOX2D& box)
```

Comment:
I do, but I'm weird I guess. I rather don't like that there's BOX2D and BOX3D at all to be honest. What we have now is more explicit, to be sure.